prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -216,6 +224,10 @@ class ReportFunction {
                 )
             )
 
+        // extract the verbose param and default to empty if not present
+        val verboseParam = request.queryParameters.getOrDefault(VERBOSE_PARAMETER, "")
+        val verbose = if (verboseParam.equals(VERBOSE_TRUE, true)) verboseParam else ""
```

Comment:
Why not make this verbose a Boolean?  There will be no need to know later what the value should be.